libs/shared/ui/src/lib/styles/components/button-action.scss

```diff
@@ -0,0 +1,67 @@
+//menu
```

Comment:
Why the name of this file is `button-action.scss` and inside we have `.btn-menu` should be `button-menu` ?